Application/registerationForm.css

```diff
@@ -0,0 +1,50 @@
+#registerationForm{
```

Comment:
Format the content of this page (you coluld use `ctrl + alt + f` shortcut in vs code to format the entire page )